gnovm/cmd/gno/internal/pkgdownload/gnopkgfetcher/gnopkgfetcher.go

```diff
@@ -0,0 +1,90 @@
+// Package gnopkgfetcher provides an implementation of [pkgdownload.PackageFetcher]
```

Comment:
should be rpcpkgfetcher

Reply:
76f3ed64b4649beff994bbfbc58039826512c455